packages/apps/haaretz.co.il/layouts/ArticleLayout.js

```diff
@@ -155,9 +150,9 @@ class ArticleLayout extends React.Component {
             // Find ArticleData content from article slot.
             // eslint-disable-next-line no-underscore-dangle
             const { authors = [], tags = [], header, } = articleData || [];
-            const authorsContentName = authors
+            const authorsContentName = authors && Array.isArray(authors)
```

Comment:
`Array.isArray` also checks if it is `null` or `undefined`, so the first check isn't needed
```suggestion
            const authorsContentName = Array.isArray(authors)
```